### PR TITLE
Ready to update links

### PR DESCRIPTION
**Pull request by Ciuffi**
We might want to wait for @kozr to fix the x-overflow bug; but this PR needs to be merged by tonight.

### CODE REVIEW

File: components/Events.vue [hunk 1]
```diff
@@ -26,10 +26,10 @@
             {{ item.date }}
           </p>
           <p class="blurb">
-            {{ item.blurb }}
+            {{ item.blurb || item.text }}
           </p>
-          <Button disabled title="Sign up" url="#" class="buttonLabel" />
-          <Button disabled title="Event page" url="#" class="buttonLabel" />
+          <Button :disabled="item.signupEnabled ? false : true" title="Sign up" :url="item.signupLink || '#'" class="buttonLabel" />
```
Comment (Eric1015) [inline components/Events.vue]: this should just be "!item.signupEnabled" if this field is a boolean

File: components/Events.vue [hunk 2]
```diff
@@ -26,10 +26,10 @@
...
+          <Button :disabled="item.signupEnabled ? false : true" title="Event page" :url="item.eventLink || '#'" class="buttonLabel" />
```
Comment (Eric1015) [inline components/Events.vue]: Here too

File: pages/index.vue [hunk 3]
```diff
@@ -7,7 +7,7 @@
         <Bulletin />
         <WhyJoin id="about" />
         <Events id="events" :items="events" />
-        <FAQ v-if="FAQFlag" id="faq" :items="FAQs" />
+        <FAQ v-if="faqFlag" id="faq" :items="FAQs" />
```
Comment (Eric1015) [inline pages/index.vue:10]: Where does this one come from?

Comment (Ciuffi) [reply pages/index.vue:10]: https://github.com/nwplus/LHD2020/blob/7c57dc460496b82e279475305ef97db09f906893/pages/index.vue#L68

**Review by AustinKobayashi — APPROVED** [review]

File: pages/index.vue (hunk 3 above)
Comment (Eric1015) [reply pages/index.vue:10]: Gotcha

**Review by acchiang — APPROVED** [review]
thanks for the hard work!

**Review by Eric1015 — APPROVED** [review]

**Pull request merged**